blackjack/game.py

```diff
@@ -0,0 +1,127 @@
+from card import Card
+from deck import Deck
+from player import Player
+from dealer import Dealer
+from interface import Interface
+import sys
+
+class Game:
+    """Run the main Blackjack game loop
+
+    Responsibilities:
+    * Starts Game
+    * Starts New Hand if Player has $
+    * Evaluate who wins and loses
+    * Pays out bets
+
+    Collaborators:
+    * Card & Deck
+    * Player & Dealer
+    * Interface for input & output
+    """
+
+    def __init__(self):
+        self.interface = Interface()
+
+    def game_start(self):
+        self.player = Player()
+        self.dealer = Dealer()
+
+
+    def start_hand(self):
+        self.player.bank -= 10
+        self.deck = Deck()
+        self.deck.shuffle_deck()
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
+        self.player.take_a_hit(self.deck)
+        self.dealer.take_a_hit(self.deck)
+        self.interface.show_current(self.player, self.dealer)
+
+    def blackjack_check(self):
+        if self.dealer.blackjack():
+            if self.player.blackjack():
+                self.interface.show_final(self.player, self.dealer)
+                self.player.bank += 10
+                self.interface.you_push(self.player)
+                #self.clear_hands()
+                return False#Hand is over
+            else:
+                self.interface.show_final(self.player, self.dealer)
+                self.interface.you_lose(self.player)
+                #self.clear_hands()
+                return False#Hand is over
+        elif self.player.blackjack():
+            self.player.bank += 25
+            self.interface.blackjack(self.player)
+            #self.clear_hands()
+            return False#Hand is over
+
+    def player_turn(self):##Come back to this
+        while True:
```

Comment:
This loop is unnecessary -- you're returning below. I've seen this a lot recently, including in your next method.